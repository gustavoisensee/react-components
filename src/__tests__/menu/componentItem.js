import React from 'react';
import renderer from 'react-test-renderer';
import MenuItem from './../../modules/menu/componentItem';

describe('MenuItem component test', () => {
  test('MenuItem component renders', () => {
    const props = {
      item: {
        id: 1,
        text: 'test',
        selected: true
      },
      click: () => {}
    };

    const tree = renderer.create(
      <MenuItem {...props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
