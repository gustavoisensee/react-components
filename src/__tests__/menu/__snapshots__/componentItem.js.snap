exports[`MenuItem component test MenuItem component renders 1`] = `
<div
  className="container">
  <input
    className="btn-selected"
    data-id={1}
    onClick={[Function]}
    type="button"
    value="test" />
</div>
`;
